docs/src/main/paradox/troubleshooting.md

```diff
@@ -0,0 +1,17 @@
+# Troubleshooting
```

Comment:
Can you make this an overview page and add a single page for that particular issue? That will help us to add a link for that particular error page later.

Reply:
:+1: done